src/Promotions/InPluginUpsells/SummerSalesBanner.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Give\Promotions\InPluginUpsells;
+
+use Give\Framework\Shims\Shim;
```

Comment:
We can remove it as there is nothing in this path.

Reply:
Resolved: https://github.com/impress-org/givewp/commit/b5deda101c5cd14a8969b7529ddd4037f19e5c47